pages/common/k6.md

```diff
@@ -0,0 +1,27 @@
+# k6
+
+> A modern load testing tool for developers and testers in the DevOps era.
```

Comment:
We can remove the "for developers and testers in the DevOps era.". We try to keep the descriptions as formal and objective as possible.

Reply:
changed in 4cbf5349